library/data/registry_entry_attachments/src/main/java/org/quiltmc/qsl/registry/attachment/impl/RegistryEntryAttachmentSync.java

```diff
@@ -77,12 +94,15 @@ public static List<PacketByteBuf> createSyncPackets() {
 		var bufs = new ArrayList<PacketByteBuf>();
 
 		for (var entry : ENCODED_VALUES_CACHE.entrySet()) {
-			for (var valueMap : entry.getValue().valueMaps()) {
+			for (var valueMap : entry.getValue().namespacesToValues()) {
```

Comment:
Either make this an explicit Pair, or use a record so that the code below isn't just "getLeft" and "getRight"